jest-preset.unit.js

```diff
@@ -3,7 +3,7 @@
 module.exports = {
   setupFilesAfterEnv: [__dirname + '/test/unit.setup.js'],
   modulePathIgnorePatterns: ['.cache', 'dist'],
-  testPathIgnorePatterns: ['.testdata.js', '.test.utils.js'],
+  testPathIgnorePatterns: ['.testdata.js', '.test.utils.[jt]s'],
```

Comment:
Not sure why this has to be done here we are not doing any ts in that PR 🤔

Reply:
Technically out of scope of this PR, but since I touched the .test.utils.js file I noticed that jest-preset.unit.js is for the whole monorepo and does include ts files (see the line below), so this would allow having ts test utils files. I can revert it if it's unwanted, no opinions on this one :)

Reply:
I tried that we doing the changes but it broke for certain package (I can't recall why I'm fine with it if that can work) :D

Reply:
I reverted it for now until we actually have a .ts to use it with :)